shared-actions/npm-setup/action.yml

```diff
@@ -0,0 +1,35 @@
+name: JS/TS environment setup
+description: 'Set up node and npm config, get repo and install its dependencies'
+inputs:
+  gh-token:
+    description: 'Github Token'
+    required: true
+  cache:
+    description: 'Use cache'
+    default: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    - name: Set up registry
+      run: |
+        npm config set '//npm.pkg.github.com/:_authToken' ${{ inputs.gh-token }}
+        npm config set @typeform:registry https://npm.pkg.github.com/
+
+    - name: Get yarn cache
+      if: inputs.cache == true
+      uses: actions/cache@v2
+      id: yarn-cache
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}-${{ hashFiles('.github/workflows/**.yml') }}
```

Comment:
hm, trying this line in my sandbox I get this. Going to dig a bit
![image](https://user-images.githubusercontent.com/6507291/111620514-1d145900-87e7-11eb-8b36-a3dd459bd284.png)